CountlyPersistency.m

```diff
@@ -28,6 +28,8 @@ @implementation CountlyPersistency
 
 NSString* const kCountlyCustomCrashLogFileName = @"CountlyCustomCrash.log";
 
+NSUInteger const CountlyRequestRemovalLoopLimit = 100;
```

Comment:
how do you miss the "k" in the start? he is still part of our team :)

Reply:
but is not "k" means key